crates/wasi/src/preview2/ctx.rs

```diff
@@ -144,8 +144,17 @@ impl WasiCtxBuilder {
         file_perms: FilePerms,
         path: impl AsRef<str>,
     ) -> &mut Self {
-        self.preopens
-            .push((Dir::new(dir, perms, file_perms), path.as_ref().to_owned()));
+        let mut open_flags = FilePerms::empty();
```

Comment:
While you're here mind adding documentation to this method? I forget how `perms` relates to `file_perms` myself

Reply:
Yep improved and documented this :)